dockerfiles/Dockerfile

```diff
@@ -65,7 +65,15 @@ COPY . ./
 # We split the builder setup out so people can target it or use as a base image without doing a full build.
 FROM builder-base AS builder
 WORKDIR /src/fluent-bit/build/
-RUN cmake -DFLB_RELEASE=On \
+
+# SIMD support: x86_64, amd64 and arm64
+ARG SIMD_CMAKE_FLAG=""
+RUN SIMD_CMAKE_FLAG="" && \
+    case "$(uname -m)" in \
```

Comment:
This won't work for containers on emulated targets - it takes the value from the host. If you try to build the container on a different host then it'll just take whatever value the host is.
Personally I think we should make this backwards compatible and just pass it in as required.

Reply:
I think we will have to update container builds to be split per target to support this as well (i.e. build and push by digest for a specific arch/target and then make a multi-arch/target manifest separately), rather than the simple current approach we have of building all on the same target: https://github.com/fluent/fluent-bit/blob/58cc2d5944a00b0844e58b91b950ecf595e05063/.github/workflows/call-build-images.yaml#L142-L158
Currently the image you make with this change would just be based on the host.

This was requested anyway to support other things like a common image for both Linux and Windows: https://github.com/fluent/fluent-bit/issues/9509